.github/workflows/main.yml

```diff
@@ -185,10 +185,31 @@ jobs:
           python2 ./update_generated.py  || exit 1
           if [ "$(git status --porcelain)" != "" ]; then echo "DataBase desync"; exit 1 ; fi
 
-      - name: Lint Samples
+      - name: Lint and Order Samples
         run: |
-          plutil -lint Docs/Sample.plist || exit 1
-          plutil -lint Docs/SampleCustom.plist || exit 1
+          OurPlistFile='Docs/Sample.plist'
+          plutil -lint "$OurPlistFile" || exit 1
+          if [ "$(git diff $OurPlistFile)" != '' ]; then
```

Comment:
This does not make sense here. Should be after the perl commands with an exit code in case of success. We do not commit the changes here, so we can only detect incorrect commits and fail the CI.